dashboard/app/models/pd/enrollment.rb

```diff
@@ -81,6 +82,13 @@ def self.for_school_district(school_district)
     joins(:school_info).where(school_infos: {school_district_id: school_district.id})
   end
 
+  scope :attended, -> {joins(:attendances).group('pd_enrollments.id')}
+  scope :not_attended, -> {includes(:attendances).where(pd_attendances: {pd_enrollment_id: nil})}
+  scope :for_ended_workshops, -> {joins(:workshop).where.not(pd_workshops: {ended_at: nil})}
```

Comment:
True, we have been using class methods everywhere in PLC land, but haven't set a standard. Even the [styleguide](https://github.com/bbatsov/rails-style-guide#named-scopes) is loose on the subject, but it seems to slightly prefer defining simple ones as scopes and more complex ones as class methods.

For consistency, I'll switch these to class methods.